.circleci/config.yml

```diff
@@ -1,9 +1,9 @@
 version: 2.1
 
 orbs:
-  win: circleci/windows@2.4.0
-  slack: circleci/slack@4.9.3
-  go: circleci/go@1.3.0
+  win: circleci/windows@5.0.0
```

Comment:
Probably a good idea to stay up-to-date with the orbs. This update eliminates py37 and changes the way the main user is treated in the vm.